example/serializers.py

```diff
@@ -206,7 +206,7 @@ class AuthorSerializer(serializers.ModelSerializer):
         self_link_view_name='author-relationships',
         queryset=AuthorBio.objects,
     )
-    entries = relations.ResourceRelatedField(
+    entries = relations.HyperlinkedRelatedField(
```

Comment:
Not so ideal to change from `ResourceRelatedField` to `HyperlinkedRelatedField` as now the one but not the other will be tested.

Hence I think it would be better to add a new  `HyperlinkedRelatedField`. Maybe comments could be used?